php-classes/Slate/CBL/Competency.php

```diff
@@ -247,17 +263,18 @@ public function getCompletionForStudent(Student $Student, $level = null, $demons
                     Demonstrations\Demonstration::$tableName,
                     $Student->ID,
                     implode(',', $this->getSkillIds()),
-                    $currentLevel,
+                    $StudentCompetency->Level,
                     ($demonstrationConditions ? 'AND ' . $demonstrationConditions . ' ' : ''),
-                    $currentLevel,
+                    $StudentCompetency->Level,
                     Skill::$tableName
                 ];
 
                 $completion = DB::oneRecord($query, $parameters);
 
                 // cast strings to floats
                 return [
-                    'currentLevel' => $currentLevel,
+                    'currentLevel' => $StudentCompetency->Level,
+                    'baselineRating' => $StudentCompetency->BaselineRating ? floatval($StudentCompetency->BaselineRating) : null,
```

Comment:
@nbey activerecord should be in charge returning fields as their correct PHP type. If `decimal` doesn't come back as a float that should be added to activerecord the same way it handles ints